CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.10.13]
+### Uncategorized
+- Add multichain tag ([#275](https://github.com/MetaMask/api-specs/pull/275))
```

Comment:
I see we haven't been categorizing these entries for a while now so I suppose we don't need to block on it here. But perhaps we should go back and categorize them at some point?